syntax/import.go

```diff
@@ -16,41 +19,108 @@ var importLocalFileVar rel.Value
 func importLocalFile() rel.Value {
 	importLocalFileOnce.Do(func() {
 		importLocalFileVar = rel.NewNativeFunction("//./", func(v rel.Value) rel.Value {
-			filepath := v.String()
-			if path.Ext(filepath) == "" {
-				filepath += ".arrai"
-			}
-			data, err := ioutil.ReadFile(filepath)
+			v, err := fileValue(v.String())
 			if err != nil {
 				panic(err)
 			}
-			expr, err := Compile(filepath, string(data))
-			if err != nil {
-				panic(err)
+
+			return v
+		})
+	})
+	return importLocalFileVar
+}
+
+var importExternalContentOnce sync.Once
+var importExternalContentVar rel.Value
+
+func importExternalContent() rel.Value {
+	importExternalContentOnce.Do(func() {
+		importExternalContentVar = rel.NewNativeFunction("//", func(v rel.Value) rel.Value {
+			importpath := v.String()
```

Comment:
I think `v` will always be a string, which is a set. If so, you should be able to call `AsString(v.(rel.Set))`.

The original was a method of `Set`, but there will be several such methods: `AsBytes`, `AsArray`, `AsDict`, and possibly more, and an open ended set of types that implement `Set`, so to avoid an MxN explosion, I decided to change it to a free function with type assertions for the few cases it supports.